src/kOS/Function/Math.cs

```diff
@@ -14,7 +15,7 @@ public override void Execute(SharedObjects shared)
             double argument = GetDouble(PopValueAssert(shared));
             AssertArgBottomAndConsume(shared);
             double result = Math.Abs(argument);
-            ReturnValue = result;
+            ReturnValue = Structure.FromPrimitive(result);
```

Comment:
Oh, I actually picked it up visually (don't have ReSharper, just CodeMaid, and that only got run on a couple of the files I reviewed).  I don't have a problem leaving it there.  To some degree I like leaving things explicit.  But at the same time, we didn't modify most of the other functions to match.  So I think I'll pull it out of these classes for the sake of consistency.